perllib/FixMyStreet/App/Controller/Admin.pm

```diff
@@ -1384,19 +1384,28 @@ sub user_add : Path('user_edit') : Args(0) {
     my $email_v = $c->get_param('email_verified');
     my $phone_v = $c->get_param('phone_verified');
 
-    unless ($email || $phone) {
-        $c->stash->{field_errors}->{username} = _('Please enter a valid email or phone number');
-    }
-    if (!$email_v && !$phone_v) {
-        $c->stash->{field_errors}->{username} = _('Please verify at least one of email/phone');
-    }
     if ($email && !is_valid_email($email)) {
         $c->stash->{field_errors}->{email} = _('Please enter a valid email');
     }
     unless ($c->get_param('name')) {
         $c->stash->{field_errors}->{name} = _('Please enter a name');
     }
 
+    if ( $c->get_param('send_login_email') ) {
+        if (!$email) {
+            $c->stash->{field_errors}->{email} = _('Please enter a valid email');
+        }
+        return if %{$c->stash->{field_errors}};
+        return $c->forward('send_login_email', { email => $email, name => $c->get_param('name') });
```

Comment:
Not sure what it does, but on the safe side, this should be `'send_login_email', [ { email => $email, name => $c->get_param('name') ] })` - second arg is an arrayref of arguments.